py/sympy/sophus/brown_conrady_camera.py

```diff
@@ -142,8 +177,25 @@ def test_codegen(self):
                 for i, line in enumerate(stream):
                     self.assertEqual(line, file_lines[i])
 
-        stream = cse_codegen(dx_pixel_from_z1_plane_x)
-        filename = "cpp_gencode/brown_conrady/dx_pixel_from_z1_plane_x.cpp"
+    def test_codegen2(self):
```

Comment:
@stevenlovegrove we have two test_codegen functions here. Was that debugging, or do we want both?

Reply:
Sorry, that was a lazy name. The methods are different - they compute different derivatives. The current Brown Conrady implementation requires derivatives both with and without the action of K. It didn't seem like the original unprojImpl was using codegen related to the methods here, so I added it as another function.